applications/lokqlDx/ViewModels/RenderingSurfaceViewModel.cs

```diff
@@ -106,6 +106,15 @@ private void DataGridCopy(string extent)
                     ClipboardAvalonia.SetText(txt);
                 }
                     break;
+
+                case "csv":
+                {
+                    var csvText = Result.EnumerateRows()
```

Comment:
Unfortunately this is an unreliable way of exporting to CSV since there is no guarantee that the objects won't have embedded commas.  I suggest using CsvSerializer instead (you can just feed it the KustoQueryResult) and saving the stream to a textwriter.